specification/azurearcdata/resource-manager/Microsoft.AzureArcData/preview/2023-11-01-preview/sqlServerAvailabilityGroups.json

```diff
@@ -0,0 +1,802 @@
+        },
+        "secondaryRoleAllowConnectionsDescription": {
+          "type": "string",
+          "description": "Whether an availability replica that is performing the secondary role (that is, a secondary replica) can accept connections from clients."
+        },
+        "replicaCreateDate": {
+          "type": "string",
+          "description": "Date that the replica was created.",
+          "format": "date-time"
+        },
+        "replicaModifyDate": {
+          "type": "string",
+          "description": "Date that the replica was modified.",
+          "format": "date-time"
+        },
+        "backupPriority": {
+          "type": "integer",
+          "description": "Represents the user-specified priority for performing backups on this replica relative to the other replicas in the same availability group.",
+          "format": "int32"
+        },
+        "readOnlyRoutingUrl": {
+          "type": "string",
+          "description": "Connectivity endpoint (URL) of the read only availability replica."
+        },
+        "readWriteRoutingUrl": {
+          "type": "string",
+          "description": "Connectivity endpoint (URL) of the read write availability replica."
+        },
+        "seedingModeDescription": {
+          "type": "string",
+          "description": "Describes seeding mode."
+        }
+      }
+    },
+    "AvailabilityGroupState": {
+      "description": "The specifications of the availability group state",
+      "type": "object",
+      "properties": {
+        "availabilityGroupReplicaRole": {
+          "type": "string",
+          "description": "Current Always On availability groups role of the availability group replica."
+        },
+        "operationalStateDescription": {
+          "type": "string",
+          "description": "Current operational state of the availability group replica"
+        },
+        "recoveryHealthDescription": {
+          "type": "string",
+          "description": "Recovery health of the availability group replica."
+        },
+        "synchronizationHealthDescription": {
+          "type": "string",
+          "description": "Reflects a rollup of the database synchronization state (synchronization_state) of all joined availability databases (also known as replicas) and the availability mode of the replica (synchronous-commit or asynchronous-commit mode). The rollup will reflect the least healthy accumulated state the databases on the replica."
+        },
+        "connectedStateDescription": {
+          "type": "string",
+          "description": "Whether a secondary replica is currently connected to the primary replica."
+        },
+        "lastConnectErrorDescription": {
+          "type": "string",
+          "description": "Text description of the last connection error of the availability group replica."
+        },
+        "lastConnectErrorTimestamp": {
```

Comment:
>lastConnectErrorTimestamp [](http://example.com/codeflow?start=9&length=25)

Consider fix to follow naming guidelines:

☑️ YOU SHOULD use an "At" suffix in names of date-time values.

For example, createdAt not created or createdDateTime.